index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <nav class="navLogo">
+      <a href="./index.html">
+        <img src="./images/navlogo.png" alt="" />
+      </a>
+      <img class="navHam" src="./images/hamburger.png" alt="" />
```

Comment:
- [ optional] only the hamburger should be inside the nav tag. the logo should be outside of it